CASMCMS-8726: Initial pass at updating docs to reflect removal of BOS v1 in CSM 1.6 (#4059)
Cray-HPE/docs-csm: operations/UAS_user_and_admin_topics/Customize_End-User_UAI_Images.md

    UAI_IMAGE_NAME=registry.local/cray/cray-uai-compute:latest
    ```

> 1. (`ncn-mw#`) Query BOS for a session template ID.

Author: ```suggestion
1. (`ncn-mw#`) Query BOS for a `sessiontemplate` ID.
```

>     Identify the session template name to use. A full list may be found with the following command:

Author: ```suggestion
    Identify the `sessiontemplate` name to use. A full list may be found with the following command:
```

    ```bash
    cray bos v2 sessiontemplates list --format yaml
    ```

    Example output:

    ```yaml
    - boot_sets:
        compute:

>           etag: d54782b3853a2d8713a597d80286b93e
>           kernel_parameters: ip=dhcp quiet spire_join_token=${SPIRE_JOIN_TOKEN}
>           node_roles_groups:
>           - Compute
>           path: s3://boot-images/0c0d4081-2e8b-433f-b6f7-e1ef0b907be3/manifest.json
>           rootfs_provider: cpss3
>           rootfs_provider_passthrough: dvs:api-gw-service-nmn.local:300:nmn0
>           type: s3
>       cfs:

Author: ```suggestion
        etag: d54782b3853a2d8713a597d80286b93e
        kernel_parameters: ip=dhcp quiet spire_join_token=${SPIRE_JOIN_TOKEN}
        node_roles_groups:
        - Compute
        path: s3://boot-images/0c0d4081-2e8b-433f-b6f7-e1ef0b907be3/manifest.json
        rootfs_provider: cpss3
        rootfs_provider_passthrough: dvs:api-gw-service-nmn.local:300:nmn0
        type: s3
    cfs:
```

        configuration: wlm-config-0.1.0

>       enable_cfs: true
>       name: wlm-sessiontemplate-0.1.0

Author: ```suggestion
    enable_cfs: true
    name: wlm-sessiontemplate-0.1.0
```

    ```

>     Alternatively, collect the session template name used during the Cray Operating System (COS) install. Refer to the "Boot COS" procedure in the COS product stream documentation.

Author: ```suggestion
    Alternatively, collect the `sessiontemplate` name used during the Cray Operating System (COS) install. Refer to the "Boot COS" procedure in the COS product stream documentation.
```

    Near the end of that procedure, the step to create a BOS session to boot the compute nodes should contain the name.

> 1. (`ncn-mw#`) Record the session template name.

Author: ```suggestion
1. (`ncn-mw#`) Record the `sessiontemplate` name.
```

    ```bash
    ST_NAME=wlm-sessiontemplate-0.1.0
    ```

1. (`ncn-mw#`) Download a compute node SquashFS.

>     Use the session template name to download a compute node SquashFS from a BOS session template name:

Author: ```suggestion
    Use the `sessiontemplate` name to download a compute node SquashFS from a BOS `sessiontemplate` name:
```
